backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Controllers/TemplateGroupController.java

```diff
@@ -24,6 +30,8 @@
 public class TemplateGroupController {
```

Comment:
In order to keep it easy to scale and make it consistent with monitors and files, try to change the RequestMapping from `/templateGroups` to `/api` and within each endpoint change it to `/templateGroups/[...]`